antora-playbook.yml

```diff
@@ -10,7 +10,15 @@ site:
   title: Couchbase Docs
   url: https://docs.couchbase.com
   start_page: home::index.adoc
-  robots: allow
+  robots: |
+    User-agent: *
+    Allow: /
+    Disallow: /sdk-api/
+    Allow: /sdk-api/*-client/
+    Allow: /sdk-api/*-core-api/
```

Comment:
should this be /*-core-io/   ?

Reply:
eek, yes indeed. And I think it's only couchbase-core-io, so lemme copy/paste that instead of being cute...
Thanks 👍